README.md

```diff
@@ -108,3 +108,51 @@ More about how the upstream action works can be found [here](https://github.com/
 ### Add new components to Release Please config file
 
 In order for components to be released, they must be in the [release-please-config.json](./release-please-config.json) file. Always ensure new components are added to this file.
+
+#### Automatically update README files with the latest version
```

Comment:
I would merge these sections somehow, and tell people what to put in the config file. It's a bit weird to have "add to this config file" and then the next section "add some more to that same file" heh

Reply:
I thought that having this as a header 3 could mean that it's optional in case someone doesn't want to do it but I think what you say is fair, I'll merge them together.